flink-connectors/flink-sql-connector-elasticsearch6/src/test/java/org/apache/flink/streaming/sql/connectors/elasticsearch6/Elasticsearch6UpsertTableSinkFactoryTest.java

```diff
@@ -65,11 +65,11 @@ public void testBuilder() {
 		final TestElasticsearch6UpsertTableSink testSink = new TestElasticsearch6UpsertTableSink(
 			false,
 			schema,
-			Collections.singletonList(new Host(HOSTNAME, PORT, SCHEMA)),
-			INDEX,
-			DOC_TYPE,
-			KEY_DELIMITER,
-			KEY_NULL_LITERAL,
+			Collections.singletonList(new Host(ElasticsearchUpsertTableSinkFactoryTestBase.HOSTNAME, ElasticsearchUpsertTableSinkFactoryTestBase.PORT, ElasticsearchUpsertTableSinkFactoryTestBase.SCHEMA)),
```

Comment:
Shouldn't this work without the `ElasticsearchUpsertTableSinkFactoryTestBase` prefix? We have `ElasticsearchUpsertTableSinkFactoryTestBase` as our base class.

Reply:
You are right. The IDE added the prefix when I moving the class files.